src/admin/components/modal/AddUserModalComponent.tsx

```diff
@@ -56,87 +62,182 @@ class AddUserModalComponent extends React.Component <any, any> {
 		return options;
 	}
 
-	componentWillReceiveProps(nextProps){}
+	makeSelectDom(optionsArr) {
+		return optionsArr.map((optionItem) =>
+			<option
+				key={optionItem} value={optionItem}>{ capitalizeFirstLetter(optionItem) }
+			</option>);
+	}
 
-	updateInputValue(event) {
-		const options = this.getCurrentInput(event.target.name);
-		options.input[event.target.name].value = event.target.value;
-		UIActions.modalWindowsChangeState(this.props.windowName, options);
+	getUserRoles() {
+		const roles = [
+			UserRole.SystemAdmin,
+			UserRole.SiteAdmin,
+			UserRole.Editor,
+			UserRole.Normal,
+		];
```

Comment:
I think it is better to use UserRole as it is, not to set them once more as an array